Fixed & updated Unity docs for road to 0.1.0 (#3603)
getsentry/sentry-docs: src/includes/getting-started-install/unity.mdx

Install the package via the [Unity Package Manager](https://docs.unity3d.com/Manual/upm-ui.html) using a Git URL to Sentry's SDK repository:

```
https://github.com/getsentry/unity.git#{{ packages.version('sentry.dotnet.unity', '0.0.5') }}
```
<Note>

> Confirm there is no trailing whitespace at the end of the URL. The Unity Package Manager will fail to find the package otherwise.

Reviewer: We've had a few issues with this, so we should keep this statement pretty explicit.
```suggestion
Confirm the URL doesn't have a trailing whitespace at the end. The Unity Package Manager will fail to find the package if a trailing whitespace is appended.
```
